### PR TITLE
remove js tab hover tracking and do it from css

### PR DESCRIPTION
**Pull request by simonxabris**
#4379 this is the issue this PR fixes.

I think this is ready to be merged, does not depend on any other work.

If there's any more work to be done, let me know and I'll fix it!

### CODE REVIEW

**Comment by simonxabris** [comment]
I think appveyor is only failing because it was disabled by this commit: https://github.com/zeit/hyper/commit/cc93bdae501263176bc96e0ed065fa5aaee13631

File: lib/components/tab.tsx [hunk 1]
```diff
@@ -88,6 +69,12 @@ export default class Tab extends React.PureComponent<TabProps, {hovered: boolean
             color: #ccc;
           }
 
+          .tab_tab:hover .tab_icon {
+            opacity: 1;
+            transform: none;
+            pointer-events: all;
+          }
+
```
Comment (LabhanshAgrawal) [inline lib/components/tab.tsx]: I think it'd be better to move this block below (just above `.tab_shape`) to it's previous location, as all the tab_icon css is there, and would better reflect the fact that the main change is from `.tab_iconHovered` to `.tab_tab:hover .tab_icon` and not the properties

Comment (simonxabris) [reply lib/components/tab.tsx]: I did it this way so all the `.tab` hover states are grouped together, but what you're suggesting also seems reasonable to me. I'll push the changes in a couple of minutes.

**Comment by Stanzilla** [comment]
Thank you!

**Pull request merged**